src/DependencyInjection/Compiler/RegisterParameterAwareReaderPass.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Iwink\GitLabWebhookBundle\DependencyInjection\Compiler;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Iwink\GitLabWebhookBundle\Annotation\ParameterAwareReader;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
+
+/**
+ * Registers a parameter aware annotation reader.
+ * @since $ver$
+ */
+class RegisterParameterAwareReaderPass implements CompilerPassInterface {
+	/**
+	 * @inheritDoc
+	 * @since $ver$
+	 */
+	public function process(ContainerBuilder $container): void {
+		try {
+			$originalDefinition = $container->findDefinition($id = 'annotations.reader');
```

Comment:
```suggestion
			$originalDefinition = $container->findDefinition('annotations.reader');
```

We shouled either remove the `$id`, or re-use it below in `setDefinition()`.